### PR TITLE
Verify duplications in the keys from secret-bootstrap's config

### PR DESCRIPTION
**Pull request by hongkailiu**
Follow up https://github.com/openshift/release/pull/12043#issuecomment-696971732

/cc @stevekuznetsov @alvaroaleman

### CODE REVIEW

**Review by alvaroaleman — APPROVED** [review]
A more generic solution for this issue would be to use https://github.com/adrienverge/yamllint which can detect this and other issues

**Comment by alvaroaleman** [comment]
/lgtm cancel
as lint doesn't pass

File: pkg/api/secretbootstrap/secretboostrap_test.go [hunk 1]
```diff
@@ -86,3 +88,69 @@ func TestResolving(t *testing.T) {
...
+			actualError := LoadConfigFromFile(filepath.Join("testdata", fmt.Sprintf("%s.yaml", t.Name())), &actual)
+			equalError(t, tc.expectedError, actualError)
+			if diff := cmp.Diff(tc.expected, actual); diff != "" {
+				t.Errorf("expected config differs from actual config: %s", diff)
+			}
+		})
+	}
+}
+
+func equalError(t *testing.T, expected, actual error) {
```
Comment (stevekuznetsov) [inline pkg/api/secretbootstrap/secretboostrap_test.go:149]: Can we stop writing a version of this function in every test file? Just inline the thing to the test method

Comment (hongkailiu) [reply pkg/api/secretbootstrap/secretboostrap_test.go]: Inlined.

**Comment by hongkailiu** [comment]
> A more generic solution for this issue would be to use https://github.com/adrienverge/yamllint which can detect this and other issues

Just found this upstream.
Maybe we can do something similar?

**Pull request merged**